instr.ml

```diff
@@ -398,12 +404,13 @@ class map = object (m)
       Assert (m#expression e)
     | Stop e ->
       Stop (m#expression e)
-    | Osr {label; cond; target; map} ->
+    | Osr {label; cond; target; var_map; frame_maps} ->
       Osr {
         label;
```

Comment:
The issue was the mismatch between the type structure and what the parser allowed, not the visitor.